accounts-db/src/accounts_db.rs

```diff
@@ -5068,7 +5094,11 @@ impl AccountsDb {
         let shrink_candidates_slots =
             std::mem::take(&mut *self.shrink_candidate_slots.lock().unwrap());
 
-        let (shrink_slots, shrink_slots_next_batch) = {
+        self.shrink_stats
```

Comment:
for some reason, this spikes to 9k after restart:
![image](https://github.com/user-attachments/assets/ff21ecd4-9592-461e-a509-ab29180ff19a)

I don't understand this yet. I think it probably needs to run a while (days).

Reply:
> I don't have an answer. Is this an anomaly?

i think we let it run over the next epoch and see what happens. I'm surprised and can't explain it.

Reply:
i think this is due to this metric:
![image](https://github.com/user-attachments/assets/144c232f-c3d3-435b-8302-d9b4ff9e525b)
This is probably due to me changing the offsets and such for ancient storages. I'll restart another node without changing offsets and see what happens. I expect this is expected behavior. A question is whether one ancient slot every 40s is sufficient to actually make it through this backlog. We will see in a few days!

Reply:
not due to any of this. I found the issue and mentioned it later.